globus_sdk/transfer/paging.py

```diff
@@ -163,22 +179,68 @@ def iterable_func(self):
 
         This method is the real workhorse of this entire module.
         """
-        # now, cap the limit per request to the max per request size
-        limit = min(self.num_results, self.max_results_per_call)
+        if not self.client_kwargs['params']:
+            self.client_kwargs['params'] = {}
+
+        # to start with, cap the limit per request to the max per request size
+        self.limit = min(self.num_results, self.max_results_per_call)
```

Comment:
Yeah, it's not documented but it should support `limit`.

I don't think it's super important that we special case this, I just thought that for cases where the result set is huge it might be nice to have a default of `100` instead of `10`.